getMoleInfo_Ensembl.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+import re
+import time
+import requests
+from Bio.Seq import Seq
+from Bio import SeqIO
+import json  # Add this import at the top of your script
+
+def getMoleculeInfo(ensembl_gene_id: str) -> dict:
+    """
+    Given an Ensembl gene ID, fetch and return information about the molecule.
+
+    Params
+    ------
+    ensembl_gene_id
+        The Ensembl gene ID to lookup.
+
+    Returns
+    -------
+    A dictionary containing information about the molecule.
+    """
+
+    ensembl_server = 'http://rest.ensembl.org'
+    endpoint = f'/lookup/id/{ensembl_gene_id}?expand=1'
+    headers = {'Content-Type': 'application/json'}
+    
+    print("Sending request to:", ensembl_server + endpoint)
+    response = requests.get(ensembl_server + endpoint, headers=headers)
+    
+    print("Response status code:", response.status_code)  # Debugging
+    if response.status_code == 200:
+        molecule_info = response.json()
+        
+        # Extract keywords
+        keywords = []
+        if 'display_name' in molecule_info:
+            keywords.append(molecule_info['display_name'])
+        if 'description' in molecule_info:
+            keywords.append(molecule_info['description'])
+        if 'synonyms' in molecule_info:
+            keywords.extend(molecule_info['synonyms'])  # Assuming synonyms is a list
+        if 'gene_family' in molecule_info:
+            keywords.append(molecule_info['gene_family']['name'])
+        
+        # Save keywords to a file
+        with open('keywords.txt', 'w') as file:
+            for keyword in keywords:
+                file.write(keyword + '\n')
+        
+        print("Molecule info retrieved:", molecule_info)  # Debugging
+        
+        try:
+            with open('molecule_info.txt', 'w') as file:
+                json.dump(molecule_info, file)  # Saves the dictionary as JSON
+            print("File written successfully.")
+        except IOError as e:
+            print("File writing error:", e)
+        
+        return molecule_info
+    else:
+        print("Failed to retrieve data - Code:", response.status_code, "Response:", response.text)
```

Comment:
woahhhh nelly. good first run. I would suggest trying to think about how you could use negative state to reduce the number of if and else's you are using. But before that make a unit test/py test to make testing things easier.  I mentioned the something and provided a small example in Analise's first PR https://github.com/Open-Access-Comp-Bio/TIMID/pull/4